<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fomulario</class>
 <widget class="QMainWindow" name="Fomulario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FORMULARIO ESTUDIANTE</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 48, 145);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>341</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(94, 255, 69);</string>
    </property>
    <property name="title">
     <string>ELIGA EL CICLO CURSADO</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>191</width>
       <height>235</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="b_ciclo1">
        <property name="styleSheet">
         <string notr="true">QPushButton{
font: 87 12pt &quot;Arial Black&quot;;
background-color:#000000ff;
color: rgb(0, 0, 0);
border-radius:5px;
border:1px sold black;
}

QPushButton:hover{
background-color:black;
color:rgb(255,255,255)
}</string>
        </property>
        <property name="text">
         <string>CICLO I</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="b_ciclo2">
        <property name="styleSheet">
         <string notr="true">QPushButton{
font: 87 12pt &quot;Arial Black&quot;;
background-color:#000000ff;
color: rgb(0, 0, 0);
border-radius:5px;
border:1px sold black;
}

QPushButton:hover{
background-color:black;
color:rgb(255,255,255)
}</string>
        </property>
        <property name="text">
         <string>CICLO II</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="b_ciclo3">
        <property name="styleSheet">
         <string notr="true">QPushButton{
font: 87 12pt &quot;Arial Black&quot;;
background-color:#000000ff;
color: rgb(0, 0, 0);
border-radius:5px;
border:1px sold black;
}

QPushButton:hover{
background-color:black;
color:rgb(255,255,255)
}</string>
        </property>
        <property name="text">
         <string>CICLO III</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="b_ciclo4">
        <property name="styleSheet">
         <string notr="true">QPushButton{
font: 87 12pt &quot;Arial Black&quot;;
background-color:#000000ff;
color: rgb(0, 0, 0);
border-radius:5px;
border:1px sold black;
}

QPushButton:hover{
background-color:black;
color:rgb(255,255,255)
}</string>
        </property>
        <property name="text">
         <string>CICLO IV</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="b_ciclo5">
        <property name="styleSheet">
         <string notr="true">QPushButton{
font: 87 12pt &quot;Arial Black&quot;;
background-color:#000000ff;
color: rgb(0, 0, 0);
border-radius:5px;
border:1px sold black;
}

QPushButton:hover{
background-color:black;
color:rgb(255,255,255)
}</string>
        </property>
        <property name="text">
         <string>CICLO V</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="b_ciclo6">
        <property name="styleSheet">
         <string notr="true">QPushButton{
font: 87 12pt &quot;Arial Black&quot;;
background-color:#000000ff;
color: rgb(0, 0, 0);
border-radius:5px;
border:1px sold black;
}

QPushButton:hover{
background-color:black;
color:rgb(255,255,255)
}</string>
        </property>
        <property name="text">
         <string>CICLO VI</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="b_salir">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>332</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 87 12pt &quot;Arial Black&quot;;
background-color:#000000ff;
color: rgb(20, 200, 220);
border-radius:5px;
border:1px sold white;
}

QPushButton:hover{
background-color:white;
}</string>
    </property>
    <property name="text">
     <string>SALIR</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
